docs/runWithDocker.md

```diff
@@ -0,0 +1,203 @@
+# Install complete Transition with Docker
+
+This installation documentation is targeted in end-users who want to try Transition locally, who do not have access to an infrastructure for web server deployment. It runs the server and its dependencies using `docker`.
+
+### 1. Install Docker for desktop
+
+Follow [the docker for desktop](https://www.docker.com/products/docker-desktop/) for instructions to install the docker for desktop application locally.
+
+If you are already familiar with docker and can run container and scripts  easily, you don't have to install docker for desktop.
+
+### 2. Prepare the docker compose file
+
+There is a few files to prepare in order to run Transition. It is suggested to create a new empty directory where to put all those file, with the following structure:
+
+```
+<base Transition dir>
+|-- docker-compose.yml
+|-- transition
+|-- |-- .env
+|-- |-- config.js
+|-- |-- polygon.geojson
+```
+
+The first file is called `docker-compose.yml` and contains the definition of the docker images to run. Copy the following code to your local `docker-compose.yml` file.
+
+```{yml}
+version: "3.7"
+# docker-compose version 1.22 is required for this version
+
+
+services:
+  transition-www:
+    image: chairemobilite/transition
+    ports:
+      - 8080:8080
+    environment:
+      PG_CONNECTION_STRING_PREFIX: postgres://postgres:pass@postgres:5432/
+      # CUSTOM Add environment variables. This can be any variable contained in the .env file. These have precedence over their value in .env
+      PROJECT_CONFIG: /config/config.js
+      TRANSITION_DOTENV: /config/.env
+    depends_on:
+      - postgres
+    tty: true
+    volumes:
+      - "./transition:/config"
+
+  postgres:
+    image: postgis/postgis
+    volumes:
+      - transition-postgis-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: pass
+
+volumes:
+  transition-postgis-data:
+```
+
+### 3. Prepare the config.js file
+
+Now create a directory named `transition` in the directory containing the `docker-compose.yml` file. This exact directory name is required as it is configured in the docker compose file.
+
+Create a file called `config.js` in the `transition` directory with the following data. You can fine-tune it to fit your need, but it would work as is.
+
+```{js}
+module.exports = {
+
+    projectShortname: 'demo_transition',
+    projectDirectory: `${__dirname}/runtime/`,
+    auth: {
+      localLogin: {
+        allowRegistration: true,
+        // This will send an email to confirm the user's email address. Email send needs to be configured. By default, users can register and directly login.
+        // confirmEmail: true
+      }
+    },
+  
+    mapDefaultCenter: {
+      lat: 45.5092960,
+      lon: -73.4769080
+    },
+  
+    languages: ['fr', 'en'],
```

Comment:
Thinking about it, we really don't need a bilingual example for a desktop setup

Reply:
At the Chaire Mobilité, given our partners and funders, though we don't need a bilingual setup, it has to be available in french. so we might as well keep it bilingual.

Reply:
Though most of the bilingualism support shouldn't be in the config.js file... But when we revisit the preferences and configuration, we can change that.

Reply:
We can provide a translation of the page in french also instead.

Reply:
En fait, il faudrati juste que l'application soit bilingue et ne pas nécessiter de config pentoute...

Reply:
Il va probablement quand même falloir traduire la page d'installation éventuellement.